apps/daimo-mobile/src/view/screen/keyRotation/DeviceScreen.tsx

```diff
@@ -101,14 +101,19 @@ export function DeviceScreen({ route, navigation }: Props) {
 
     async function removeKey() {
       console.log(`[DEVICE] Removing device ${devicePubkey}`);
-      await exec();
+      exec();
```

Comment:
small but `removeKey` is no longer async now ^

Reply:
fixed